sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -0,0 +1,312 @@
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+    AND m.metrics.string.metrics_install_source IS NOT NULL
+    AND m.metrics.string.metrics_adjust_network IS NOT NULL
+  GROUP BY
+    client_id
+  UNION ALL
+  -- Firefox Preview nightly
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(submission_timestamp) AS min_submission_timestamp,
+    ARRAY_AGG(metrics.string.metrics_adjust_network ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS adjust_network,
+    ARRAY_AGG(metrics.string.metrics_install_source ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS install_source
+  FROM
+    org_mozilla_fenix_nightly.metrics AS m
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+    AND m.metrics.string.metrics_install_source IS NOT NULL
+    AND m.metrics.string.metrics_adjust_network IS NOT NULL
+  GROUP BY
+    client_id
+  UNION ALL
+  -- Fenix nightly
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(submission_timestamp) AS min_submission_timestamp,
+    ARRAY_AGG(metrics.string.metrics_adjust_network ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS adjust_network,
+    ARRAY_AGG(metrics.string.metrics_install_source ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS install_source
+  FROM
+    org_mozilla_fennec_aurora.metrics AS m
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+    AND m.metrics.string.metrics_install_source IS NOT NULL
+    AND m.metrics.string.metrics_adjust_network IS NOT NULL
+  GROUP BY
+    client_id
+  UNION ALL
+  -- Fenix beta
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(submission_timestamp) AS min_submission_timestamp,
+    ARRAY_AGG(metrics.string.metrics_adjust_network ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS adjust_network,
+    ARRAY_AGG(metrics.string.metrics_install_source ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS install_source
+  FROM
+    org_mozilla_firefox_beta.metrics AS m
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+    AND m.metrics.string.metrics_install_source IS NOT NULL
+    AND m.metrics.string.metrics_adjust_network IS NOT NULL
+  GROUP BY
+    client_id
+  UNION ALL
+  -- Fenix Release
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(submission_timestamp) AS min_submission_timestamp,
+    ARRAY_AGG(metrics.string.metrics_adjust_network IGNORE NULLS ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS adjust_network,
+    ARRAY_AGG(metrics.string.metrics_install_source IGNORE NULLS ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS install_source
+  FROM
+    org_mozilla_firefox.metrics AS m
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+  GROUP BY
+    client_id
+),
+-- Get first adjust data per client_id per date
+metrics_unique_attribution AS (
+  SELECT
+    client_id,
+    DATE(min_submission_timestamp) AS metrics_ping_submission_date,
```

Comment:
Required as a timestamp to be able to compare which ping comes first btw. metrics and first_session.

Reply:
Updated to datetime.